app/routes/overview/components/PublicFrontpage.js

```diff
@@ -81,13 +81,13 @@ class PublicFrontpage extends Component<Props, State> {
 
     return (
       <Container>
-        {/*<Banner
-          header="Velkommen alle nye studenter!"
-          subHeader="Trykk her for informasjon om fadderperioden"
-          internal
-          link="/articles/359"
-          color={COLORS.lightBlue}
-        />*/}
+        <Banner
+          header=" Velkommen til fadderperioden 2022!"
```

Comment:
Is this space intented?

Reply:
THERE NEVER WAS ANY SPACE! 😬

Reply:
<img width="922" alt="image" src="https://user-images.githubusercontent.com/23152018/174808757-20c30a9d-ee74-4d71-a5ae-1909fde6dcbd.png">
😬